outputs.tf

```
@@ -21,5 +21,5 @@ output "log_group" {
 
 output "kms_key_id" {
   description = "The KMS Key ARN used to encrypt the logs"
-  value       = length(aws_cloudtrail.this) > 0 ? aws_cloudtrail.this[0].kms_key_id : null
+  value       = length(aws_cloudtrail.this) > 0 ? aws_cloudtrail.this.kms_key_id : null
```

Since `aws_cloudtrail.this` is no longer optional, this whole construct can be simplifed:

```suggestion
  value       = aws_cloudtrail.this.kms_key_id
```

---

Ah yes.  Good call.